frontend/src/model/ModelReleaseDisplay.tsx

```diff
@@ -110,7 +110,7 @@ export default function ModelReleaseDisplay({
                   spacing={2}
                 >
                   <Link href='/beta'>{file}</Link>
-                  <Typography variant='caption'>123GB</Typography>
+                  {/* <Typography variant='caption'>123GB</Typography> */}
```

Comment:
Commented out code

Reply:
This is being kept in until we know how we're going to handle sizes for artefacts (they're just returned as strings), I'll make a todo comment.